newIDE/app/src/UI/SortableVirtualizedItemList/ItemRow.js

```diff
@@ -27,8 +27,9 @@ type Props = {
   getThumbnail?: () => string,
   selected: true,
   onItemSelected: () => void,
+  buildMenuTemplate: Item => any,
```

Comment:
This seems to be not related to this PR, and probably and copy-paste of an old version of the file. Can you put `buildMenuTemplate: () => any,` back again?

Reply:
i just noticed that I have it twice :o looks like bad resolve of conflicts. Fixed it now